docs/pages/index.js

```diff
@@ -1,3 +1,5 @@
-import LandingPage from '@material-ui/monorepo/docs/pages/index';
+import * as React from 'react';
 
-export default LandingPage;
+export default function Home() {
+  return <div>/</div>;
```

Comment:
In #2465 I added the missing dependencies to make the home to open without errors.

Reply:
Did I remove the need for these extra dependencies? O:)